docsite/rst/developing_modules.rst

```diff
@@ -200,6 +200,24 @@ This should return something like::
 
     {"changed": true, "time": "2012-03-14 12:23:00.000307"}
 
+.. _binary_module_reading_input:
+
+Binary Modules Input
+~~~~~~~~~~~~~~~~~~~~
+
+Support for binary modules was added in Ansible 2.2.  When Ansible detects a binary module, it will proceed to
+supply the argument input as a file on ``argv[1]`` that is formatted as JSON.  The JSON contents of that file
+would resemble something similar to::
+
+    {
+        "name": "Ansible",
```

Comment:
Maybe use something different than name and Ansible here to be clear that this the module parameters rather than more special variables.  Maybe something like "data" like the ping module uses?